api-key-thing-python/Generator.py

```
@@ -0,0 +1,25 @@
+
+k = f"sk_kent_{secrets.token_hex(32)}"
+db = get_db()
+n = datetime.now(timezone.utc)
+
+db.append({
+    'id': int(n.timestamp() * 1000), 
+    'hash': h(k), 
+    'createdAt': n.strftime('%Y-%m-%dT%H:%M:%S.%f')[:-3] + 'Z'
+})
+
+with open(p, 'w') as f: json.dump(db, f, indent=2)
```

Ruff (0.15.7)

[error] Multiple statements on one line (colon)

(E701)